jbmc/src/java_bytecode/java_bytecode_convert_class.h

```diff
@@ -41,6 +41,10 @@ void mark_java_implicitly_generic_class_type(
   const irep_idt &class_name,
   symbol_tablet &symbol_table);
 
+/// Register in the \p symbol_table new symbols for the objects
+/// java::array[X] where X is byte, float, int, char...
+void add_array_types(symbol_tablet &symbol_table);
```

Comment:
That's quite a general name in the global namespace. Maybe `add_java_array_types`?

Reply:
Or `add_java_primitive_array_types`?

Reply:
@allredj also adds `java::array[reference]` so that name would be inaccurate.

Reply:
Oh right. Even more reason to document properly! ;-)